scripts/build.sh

```
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+cd dash-sdk-bindings
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)